exampleCourse/questions/demoAutograderSquare/info.json

```diff
@@ -0,0 +1,21 @@
+{
+    "uuid": "994D6DDE-B41F-4EC5-90CC-9CE18EF5558B",
+    "title": "Autograder demo: square a number",
```

Comment:
The other exampleCourse question titles are (almost all) using the convention of a capital letter after the colon, so this would be `Autograder demo: Square a number`, and similarly for the other questions.